### DISCUSSION TITLE
Which Font

### DISCUSSION THREAD

**Discussion by gamagan** [ocornut/imgui#4926] [New Users Build/Link/Run issues ONLY!]
Which font is this?
https://user-images.githubusercontent.com/8225057/81696852-c15d9e80-9464-11ea-9cab-2a4d4fc84396.jpg

Mentioned in this font tutorial:
https://github.com/ocornut/imgui/blob/master/docs/FONTS.md

The font is used by this app:
https://www.avoyd.com/

That font is nice and clean for an UI.

**Comment by PathogenDavid** [comment]
I'm pretty sure that's just [Noto Sans](https://fonts.google.com/noto/specimen/Noto%20Sans). (It's referenced earlier in the fonts guide.)
